code-challenges.js

```diff
@@ -20,7 +20,24 @@ const fruit4 = "kiwi"
 
 // --------------------1) Create a function that takes in a number and determines if the given number is below boiling point, at boiling point, or above boiling point. Boiling point is 212 degrees Fahrenheit. Use the test variables provided below.
 
-// Pseudo code:
+// Pseudo code: 
+// Process
+// create a function that will take a value called temperature.
+const boilingPoint = (temperature) => {
+    // if the temperature is less than 212 return "is below boiling point"
+    if (temperature < 212){
+        return `${temperature} is below boiling point`
+    }
+    // if the temperatue is above 212 return "is above boiling point"
+    else if (temperature > 212){
+        return `${temperature} is above boiling point`
+    }
+    // if the temperature is equal to 212 return "is at boiling point"
+    else {
+        return `${temperature} is at boiling point`
+    }
+}
+
```

Comment:
Great job using string interpolation. One thing to consider, what if you received a string as an argument? What would happen?